src/components/Container/TodoList.js

```diff
@@ -0,0 +1,236 @@
+import { useContext, useRef } from "react";
+import Context from "./store/Context";
+import { FiMessageSquare } from "react-icons/fi";
+import { AiOutlinePlus } from "react-icons/ai";
+import { AiOutlineCheck } from "react-icons/ai";
+import { FaTimes } from "react-icons/fa";
+import { setTodoInput, createTodos } from "./store/actions";
+import {
+  StyledTodoList,
+  StyledList,
+  HeadContent,
+  Content,
+  Icons,
+  InputBtn,
+  InputSection,
+  MonitorBtn,
+} from "./TodoList.styled";
+
+function TodoList() {
+  const [state, dispatch] = useContext(Context);
+  const { todoInput, todos, isInputExpand } = state;
+  console.log(isInputExpand);
+  const handleSubmit = () => {
+    dispatch(createTodos(todoInput));
+    dispatch(setTodoInput(""));
+    nameRef.current.focus();
+  };
+  const nameRef = useRef();
+  return (
+    <StyledTodoList>
+      <StyledList>
+        <HeadContent>Check List</HeadContent>
+        <ul>
+          {todos.map((todo, index) => {
+            return (
+              <Content key={index}>
```

Comment:
không nên dùng index làm key Đức: https://stackoverflow.com/questions/59517962/react-using-index-as-key-for-items-in-the-list